.github/workflows/build.yml

```diff
@@ -213,3 +214,42 @@ jobs:
         yarn storybook:deploy
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  publish:
+      runs-on: ubuntu-latest
```

Comment:
you _should_ be able to do:


`needs: [tests-atoms, tests-molecules, .....]` 

etc to ensure that all test jobs have completed before publishing 👍🏼